checkout.ps1

```diff
@@ -1,13 +1,20 @@
+Write-Output "Current dir: $pwd"
+Get-ChildItem "$path" -Recurse -Directory
 if(Test-Path "playbooks") {
-    Remove-Item -Recurse "playbooks"
+    Write-Output "remove playbooks directory"
+    Get-ChildItem "playbooks" -Recurse | Remove-Item -Recurse -Force
+    Start-Sleep -s 2
+    Get-Item "playbooks" | Remove-Item -Recurse -Force
+    Start-Sleep -s 2
 }
 
+Write-Output "clone Repo"
 if($env:GITHUB_EVENT_NAME -match "pull_request") {
-    (git clone https://github.com/${GITHUB_ACTOR}/tutorials.git playbooks) -or (git clone https://github.com/devonfw-forge/tutorials.git playbooks)
+    (git clone https://github.com/$env:GITHUB_ACTOR/tutorials.git playbooks) -or (git clone https://github.com/devonfw-forge/tutorials.git playbooks)
```

Comment:
Could we somehow make devonfw-forge/tutorials configurable by a single constant to Properly end up in maintainable code in case the repo name or the organization changes?

Reply:
I've added a config file where the url to the repo is configurable.